Apps/phhackerone/hackerone.json

```diff
@@ -0,0 +1,647 @@
+                },
+                {
+                    "data_path": "action_result.data.*.reports",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.data.*.count",
+                    "data_type": "numeric",
+                    "column_name": "Count",
+                    "column_order": 1
+                },
+                {
+                    "data_path": "action_result.status",
+                    "data_type": "string",
+                    "example_values": [
+                        "success",
+                        "failed"
+                    ],
+                    "column_name": "Status",
+                    "column_order": 0
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "summary.total_objects",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1
+                    ]
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1
+                    ]
+                }
+            ],
+            "render": {
+                "type": "table"
+            },
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "update tracking id",
+            "description": "Update HackerOne report with tracking ID",
+            "type": "generic",
+            "identifier": "update_id",
+            "read_only": false,
+            "parameters": {
+                "report_id": {
+                    "description": "ID of report to update",
+                    "data_type": "string",
+                    "order": 0,
+                    "required": true,
+                    "primary": true,
+                    "contains": [
+                        "hackerone report id"
+                    ]
+                },
+                "tracking_id": {
+                    "description": "ID of tracking ticket",
+                    "data_type": "string",
+                    "order": 1,
+                    "required": true
+                }
+            },
+            "output": [{
+                    "data_path": "action_result.parameter.report_id",
+                    "data_type": "string",
+                    "contains": [
+                        "hackerone report id"
+                    ],
+                    "column_name": "Report ID",
+                    "column_order": 1
+                },
+                {
+                    "data_path": "action_result.parameter.tracking_id",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.data.*",
```

Comment:
Should there be a final key after `data.*`?

Reply:
Hi @alexa-phantom,
It should be there, but we got the same from the contributor. And, it is a community app, so we are not able to identify what it should be. Please suggest what we should do in a such case. Should we keep as it is or removes it?

Reply:
Let's remove it. Thanks.